kubernetes/create-config-maps.sh.j2

```diff
@@ -20,6 +20,14 @@ kubectl create secret generic concent-api-secrets            \
     --from-file=secrets.py=concent-secrets/secrets.py        \
     --from-literal=__init__.py=
 
+{% if gke.cluster == "concent-dev" or gke.cluster == "concent-staging" %}
+kubectl create secret generic signing-service-secrets                      \
+    --from-file=ethereum-secrets.py=concent-secrets/ethereum-secrets.py    \
+    --from-literal=__init__.py=
```

Comment:
Let's not hard-code stuff like this by cluster names. This way you won't see this difference when looking at cluster's config file.

Please add a variable called `signing_service_on_the_cluster` in `var.yml` and set it to `true` for `dev` and `staging`.

Reply:
Done.